nodejs/express/books/package.json

```diff
@@ -5,21 +5,23 @@
   "main": "app.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "setup": "node createBooksTable.js",
-    "start": "node app.js"
+    "start": "node server.js"
   },
   "author": "Abdirahim Musse",
   "license": "MIT",
   "dependencies": {
-    "body-parser": "^1.18.3",
+    "body-parser": "^1.19.0",
     "connect-flash": "^0.1.1",
+    "cookie-parser": "^1.4.5",
+    "crypto": "^1.0.1",
     "ejs": "^2.6.1",
     "express": "^4.16.3",
     "express-session": "^1.15.6",
-    "idb-pconnector": "^1.0.2",
     "passport": "^0.4.0",
     "passport-local": "^1.0.0",
-    "appmetrics-dash": "^4.0.0"
+    "odbc": "alpha",
```

Comment:
I don't know if I have seen this, does it just pull from the `alpha` tag on npm? I just within the last few days published the latest and greatest to `beta`, so it might be better to include that. And I am hoping beyond all hope that I can get `2.4.0` out the door this week on `latest` branch.

Reply:
Changed to beta